content/index.html

```diff
@@ -343,7 +343,11 @@ <h2 class="white py-small">Get started with Cassandra, fast.</h2>
 	</body>
 	<script>
         jQuery(function(){
+<<<<<<< HEAD
             var imgArr = ['ably_full_logo.png','activision_black_text.png','adstage_full_white_text.jpg','bazaar_full.png','best_buy.png','bigmate.png','blackberry_black_text.jpg','blackrock_logo.png','bloomberg.png','campaign-monitor_black_text.png','cern_blue_text.png','clearcapital_full.png','cloudkick.png','cloudtrax_logo.png','comcast_logo.png','constant_contact_white_text.png','coursera_blue_text.png','datacloud.png','discord_full.png','ebay_logo.png','equinix.png','flant.png','grubhub_red_text.jpg','home_depot.jpg','hulu_white_background.png','ibm_black_text.svg','instaclustr.png','instagram.jpg','instana_full.png','intuit_turbotax_full.png','keen.png','locstat_full.png','macquarie_bank_black_text.jpg','macys_black_text.png','maths_pathway_full.jpg','metro_nom.png','mobile_pay.png','monzo_full.png','netflix.png','new_york_times.png','nhn_techorus.png','ooyala_vertical.png','outbrain_full.png','paidy.png','penn_mutual.jpg','protectwise.png','revrtrax.png','sky_logo.png','spotify_full.png','target_full.png','uber_black_text.jpg','urban_airship.jpg','walmart.png','woods_hole_oceanographic.png','yelp.jpg'];
+=======
+            var imgArr = ['ably_full_logo.png','activision_black_text.png','adstage_full_white_text.jpg','bazaar_full.png','best_buy.png','bigmate.png','blackberry_black_text.jpg','blackrock_logo.png','bloomberg.png','campaign-monitor_black_text.png','cern_blue_text.png','clearcapital_full.png','cloudkick.png','cloudtrax_logo.png','comcast_logo.png','constant_contact_white_text.png','coursera_blue_text.png','datacloud.png','discord_full.png','ebay_logo.png','equinix.png','flant.png','grubhub_red_text.jpg','home_depot.jpg','hornet.png','hulu_white_background.png','ibm_black_text.svg','instaclustr.png','instagram.png','instana_full.png','intuit_turbotax_full.png','keen.png','locstat_full.png','macquarie_bank_black_text.jpg','macys_black_text.png','maths_pathway_full.jpg','metro_nom.png','mobile_pay.png','monzo_full.png','netflix.png','new_york_times.png','nhn_techorus.png','ooyala_vertical.png','outbrain_full.png','paidy.png','penn_mutual.jpg','protectwise.png','revrtrax.png','sky_logo.png','spotify_full.png','target_full.png','uber_black_text.jpg','urban_airship.jpg','walmart.png','woods_hole_oceanographic.png','yelp.jpg'];
+>>>>>>> CASSANDRA-16715 Updated website with content for June 2021
```

Comment:
@ErickRamirezAU this needs to be resolved^

Reply:
@michaelsembwever there were 2 changes to that list -- added `hornet.png` (with new image included in `img/companies/`) and replaced `instagram.jpg` with `instagram.png` (also in `img/companies/`). Is that what you meant?

Reply:
Fixed with 02979d4.